test/e2e/mco_test.go

```diff
@@ -92,3 +94,26 @@ func waitForAllMastersUpdate(cs *framework.ClientSet, mastersSchedulable bool) e
 		return true, nil
 	})
 }
+
+func TestClusterOperatorStatusExtension(t *testing.T) {
```

Comment:
this was broken in master/4.5/4.4 until https://github.com/openshift/cluster-config-operator/pull/136 and since there was no gating on PRs breaking the Status.Extension field, this test will do it since we rely on that field to be set when something fails in the MCO

Reply:
To make sure, if someone else breaks the extension field, it'll show up in our e2e test as a failure? I feel like we should be gating it somewhere else as well

Reply:
yep, working on having a proper test but still under discussion - this is the fastest way to get something in - what we care anyway is about having this field work so whoever breaks it, it breaks us so it kind of make sense that we test our logic (we're not testing the field per-se)